.github/buildomat/build-and-test.sh

```diff
@@ -95,9 +95,16 @@ ptime -m timeout 1h cargo test --doc --locked --verbose --no-fail-fast
 # Build the live-tests.  This is only supported on illumos.
 # We also can't actually run them here.  See the README for more details.
 if [[ $target_os == "illumos" ]]; then
+    banner "live-test build"
```

Comment:
![Screenshot from 2024-09-19 13-15-40](https://github.com/user-attachments/assets/42a76ced-f205-40ef-a8a5-6f47f101f04d)

The "build" gets dropped. (That's because `banner` only shows the first 10 characters, per the man page.)